common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -1,6 +1,6 @@
 import { css } from "@emotion/react";
-import { neutral, from, space, headline } from '@guardian/source-foundations';
-import { FirstPublished } from './FirstPublished';
+import { neutral, from, space, headline } from "@guardian/source-foundations";
+import { FirstPublished } from "./FirstPublished";
```

Comment:
I think we lack any code-style checks in common-rendering which is why we're seeing changes like this (which isn't ideal) - perhaps something we need to look at outside this team

Reply:
I see, I think this is auto-formatting that I hadn't noticed then. If it's simpler I can remove the formatting changes from this PR and possibly address them separately (run prettier in common-rendering in a separate PR maybe?)

Reply:
I think it's fine to keep them here. They're small and self contained. We're working on addressing the linting issues for common in a separate thread

Reply:
Alright, sounds good!